src/applications/gi/containers/GiBillApp.jsx

```diff
@@ -113,7 +121,7 @@ export class GiBillApp extends React.Component {
             </DowntimeNotification>
             <AboutThisTool />
             <Disclaimer />
-            <Modals />
+            <Modals gibctCh33BenefitRateUpdate={gibctCh33BenefitRateUpdate} />
```

Comment:
```suggestion
            <Modals gibctCh33BenefitRateUpdate={featureFlags} />
```

Reply:
What about passing all the feature flags down into the Modals component so they are accessible if we need them for future modals?